pkg/lib/config/secret.go

```diff
@@ -174,7 +174,7 @@ const (
 	SMTPServerCredentialsKey  SecretKey = "mail.smtp"
 	TwilioCredentialsKey      SecretKey = "sms.twilio"
 	NexmoCredentialsKey       SecretKey = "sms.nexmo"
-	OIDCKeyMaterialsKey       SecretKey = "oidc"
+	OAuthKeyMaterialsKey      SecretKey = "oauth"
```

Comment:
This would break existing apps. Can we avoid renaming?

Reply:
Sure we can keep the name, however we will be stuck with an inaccurate name thereafter.